bin/parse_web_docs.dart

```diff
@@ -0,0 +1,73 @@
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:path/path.dart';
+import 'package:solid_lints/utils/docs_parser/output_formatters/docusaurus_formatter.dart';
+import 'package:solid_lints/utils/docs_parser/parsers/docs_parser.dart';
+
+void main(List<String> rawArgs) async {
+  final readmeDefaultPath = join(Directory.current.parent.path, 'README.md');
+  final docusaurusDefaultPath = join(
+    Directory.current.parent.path,
+    'doc',
+    'docusaurus',
+    'docs',
+  );
+
+  final argsParser = ArgParser()
+    ..addOption(
+      'path',
+      abbr: 'p',
+      help: 'Path to the directory containing lint rules',
+      mandatory: true,
+    )
+    ..addOption(
+      'docs-dir',
+      abbr: 'o',
+      help: 'Parser output path. i.e "docusaurus/docs" directory',
+      defaultsTo: docusaurusDefaultPath,
+    )
+    ..addOption(
+      'output-dir',
```

Comment:
```suggestion
      'output-dirname',
```

Reply:
Scratch it -- just remove the argument and make the path part of the `docs-dir` argument.

Reply:
it's necessary for separating outer folder which contains `intro.md` file and the inner folder with lints documenation

Reply:
but, after thinking a little, i think we could resolve path for `intro.md` file by just using parent directory of output dir. 
will change it tomorrow